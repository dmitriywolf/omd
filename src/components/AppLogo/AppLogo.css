.app__logo {
  margin: 0;
  font-size: 30px;
  color: #ffffff;
}
