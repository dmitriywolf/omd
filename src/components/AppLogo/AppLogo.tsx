import React from 'react';
import './AppLogo.css';

const AppLogo = () => {
  return (
    <h2 className="app__logo">OMDb API Movies</h2>
  )
};

export default AppLogo;
