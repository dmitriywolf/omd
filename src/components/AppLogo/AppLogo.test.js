import React from 'react';
import AppLogo from './AppLogo';
import {shallow} from 'enzyme';
import toJson from "enzyme-to-json";

describe("AppLogo component:", () => {
  it("should render AppLogo component", () => {
    const component = shallow(<AppLogo/>);
    expect(toJson(component)).toMatchSnapshot();
  })
});
