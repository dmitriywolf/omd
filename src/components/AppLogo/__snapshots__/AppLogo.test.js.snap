// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppLogo component: should render AppLogo component 1`] = `
<h2
  className="app__logo"
>
  OMDb API Movies
</h2>
`;
